SimTKcommon/Geometry/src/DecorativeGeometryRep.h

```diff
@@ -587,7 +595,7 @@ void implementGeometry(DecorativeGeometryImplementation& geometry) const overrid
 SimTK_DOWNCAST(DecorativeMeshFileRep, DecorativeGeometryRep);
 private:
 std::string meshFile;
-
+std::shared_ptr<PolygonalMesh> pmesh;
```

Comment:
Since PolygonalMesh has shallow copy semantics I don't think this should be a pointer at all. Can't it just be a PolygonalMesh as in DecorativeMesh? You could have a separate "isInitialized" flag for the lazy load.